macros.d/macros.fjava

```diff
@@ -491,3 +492,32 @@ sort -u -o ${_filelist} ${_filelist} \
 
 %pom_change_dep(rf)     %{expand: %{__pom_call}change_dep    %**
 }
+
+# %generate_javadoc_package - generate javadoc subpackage definition
```

Comment:
What about just %javadoc_package? RPM sections are in declarative not imperative language - there is %package, not %define_package, %files, not %declare_files and so on.